vmdk_plugin/utils/volumeProperties/VolumePropertiesUtil.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This util provides various helper methods that can be used by testcases to
+// fetch information like capacity, disk-format and vm-attched field
+// for volume using docker cli or admin cli.
+
+package e2e
+
+import (
+	"fmt"
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"strings"
+)
+
+var ADMIN_CLI_LS = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls "
+var DOCKER_CLI_INSPC = "docker volume inspect "
+
+// returns attached to vm field of volume using docker cli
+func GetVmAttachedToVolUsingDockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{index .Status \"created by VM\"}}' " + volName
+	op := ExecCmd(hostname, cmd)
+	return strings.TrimSpace(string(op))
+}
+
+// returns attached to vm field of volume using admin cli
+func GetVmAttachedToVolUsingAdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_LS + "-c volume,attached-to 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	return string(volProps[1])
```

Comment:
This raises a serious concern by taking account of the common code...

some thoughts
1. what happens op is null or empty i.e. the asked volume not found .. this util will not be useful 
2. let's say there is a bug in the system and return duplicates .. this approach is dangerous as it is returning 1st index.